src/Symfony/Component/Console/Input/InputArgument.php

```diff
@@ -111,6 +113,19 @@ public function getDefault(): string|bool|int|float|array|null
         return $this->default;
     }
 
+    public function setCompletionValues(array|\Closure $completionValues = null)
```

Comment:
we should document the signature of the closure somewhere for clarity

Reply:
actually, let's remove this setter and keep only the constructor argument

Reply:
I used the psalm syntax to document the Closure. https://psalm.dev/docs/annotating_code/type_syntax/callable_types/